Pokedex/ProfileViewController.swift

```diff
@@ -0,0 +1,296 @@
+//
+//  ProfileViewController.swift
+//  Pokedex
+//
+//  Created by Shubham Goenka on 29/09/16.
+//  Copyright © 2016 trainingprogram. All rights reserved.
+//
+
+import UIKit
+
+var nameAndNumberView: NameAndNumberView?
+let testPokemon = PokemonGenerator.getPokemonArray()[0]
+
+
+class ProfileViewController: UIViewController, UIScrollViewDelegate {
+    var passedPokemon: Pokemon?
+    var scrollView: UIScrollView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+        view.backgroundColor = UIColor(colorLiteralRed: 100/255, green: 0, blue: 0, alpha: 1)
+        setUI()
+    }
+    
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+    
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        scrollView.frame = view.bounds
+        scrollView.isScrollEnabled = true
+        scrollView.contentSize = view.frame.size
+
+
+    }
+    
+    func setUI() {
+        
+        //setups scrollview
+        scrollView = UIScrollView()
+        scrollView.delegate = self
+        view.addSubview(scrollView)
+        
+        var containerView = UIView()
+        scrollView.addSubview(containerView)
```

Comment:
i can't scroll down to add to favorites which can be fixed with following code:

scrollView.setFrame, you have to account for the fact that you have a navigation tab on the bottom!